python/pyspark/sql/dataframe.py

```diff
@@ -732,6 +732,11 @@ def repartitionByRange(self, numPartitions, *cols):
         At least one partition-by expression must be specified.
         When no explicit sort order is specified, "ascending nulls first" is assumed.
 
+        Note that due to performance reasons this method uses sampling to estimate the ranges.
```

Comment:
Besides Python, we also have `repartitionByRange` API in R. Can you also update it?

Reply:
Oh right, I missed it! Pushed.